Translate Cars.com and Dynatrace join the Foundation to support the stability and success of the Node.js platform (#414)

devjin0617 opened the pull request:
많이 부족하지만 최대한 어색하지 않도록 진행해봤습니다.

리뷰 부탁드리겠습니다.

devjin0617 committed:
Merge pull request #1 from nodejs/master

update repo

devjin0617 committed:
Translate cars-dynatrace.md (#414)

devjin0617 committed:
Update cars-dynatrace.md (#414)

taggon commented on source/_posts/2016-08-17-announcements-cars-dynatrace.md:
```diff
@@ -0,0 +1,140 @@
+-->
+
+**샌프란시스코, 2016년 8월 17일**
+Node.js 플랫폼 발전을 위해 지역사회 주도 기반의 컨소시엄인 [Node.js 재단](https://nodejs.org/en/foundation/)은 오늘 [Cars.com](https://www.cars.com/)과 [Dynatrace](https://www.dynatrace.com)가 재단에 실버회원으로 가입했습니다.
```
"a community-led ... Node.js platform," 부분은 거의 템플릿입니다. Announcement에 항상 나오는 부분이라 다른 파일(예 #439)을 참고하시면 조금 편합니다.

outsideris commented on source/_posts/2016-08-17-announcements-cars-dynatrace.md:
```diff
@@ -0,0 +1,140 @@
+developer survey.
+-->
+
+기업용 도구 및 서비스 에코시스템은 Node.js를 중심으로 빠르게 융합되어 엔터프라이즈 애플리케이션, 로봇, API 엔진, 클라우드 스택, 모바일 웹사이트의 모든 것을 지원하는 범용 플랫폼으로 발전되었습니다. 자바스크립트 런타임 환경은 자원 효율, 성능이 뛰어나며 확장성에 잘 어울려 애플리케이션 개발 및 마이크로 서비스 아키텍처를 위해 스타트업, 엔터프라이즈 기업에서 점차 대체되고 있습니다. 또한 Node.js는 StackOverflow가 조사한 2016년 개발자 설문조사에서 풀스택, 프론트엔드 및 맥엔드 개발자를 위한 상위 10개 언어 중 하나입니다.
```
`맥엔드` -> 백엔드

outsideris commented on source/_posts/2016-08-17-announcements-cars-dynatrace.md:
```diff
@@ -0,0 +1,140 @@
+More about today’s new Node.js Foundation members:
+-->
+
+오늘의 새로운 Node.js 재단 멤버에 대해 자세히 알아보십시오:
```
오늘보다는 "이번" 등으로 의역하는건 어떨까요?

"이번의 새 Node.js 재단 멤버에 대해 자세히 알아보겠습니다."

outsideris commented on source/_posts/2016-08-17-announcements-cars-dynatrace.md:
```diff
@@ -0,0 +1,140 @@
+support the needs and demands of enterprise users.”
+-->
+
+Node.js 재단 커뮤니티 관리자인 Mikeal Rogers는 "Node.js 재단에 합류함으로써 기업들은 안정성의 안정성을 높이고 광범위한 채택과 개발을 가속화하는 데 일조하고 있습니다."
```
문장이 종료가 안되서 마지막에 "라고 말했습니다"가 추가되어야 할 것 같습니다.

outsideris reviewed (commented):
다 봤습니다.

devjin0617 committed:
Update cars-dynatrace.md (#414)

맥엔드 → 백엔드
오늘의 새로운 Node.js 재단 멤버에 대해 자세히 알아보십시오 → 이번의 새 Node.js 재단 멤버에 대해 자세히 알아보겠습니다.
문장종료시 "라고 말했습니다." 추가

taggon commented on source/_posts/2016-08-17-announcements-cars-dynatrace.md:
```diff
@@ -0,0 +1,141 @@
+More about today’s new Node.js Foundation members:
+-->
+
+이번의 새 Node.js 재단 멤버에 대해 자세히 알아보겠습니다:
```
한국어에서는 문장 말미에 콜론(:)을 쓰지 않습니다. 마침표로 바꾸는 게 좋을 것 같습니다.

taggon commented on source/_posts/2016-08-17-announcements-cars-dynatrace.md:
```diff
@@ -0,0 +1,141 @@
+-->
+
+Node.js 재단 커뮤니티 관리자인 Mikeal Rogers는
+"Node.js 재단에 합류함으로써 기업들은 안정성의 안정성을 높이고 광범위한 채택과 개발을 가속화하는 데 일조하고 있습니다."
```
안전성의 안정성 <- 중복됐습니다.

yous replied:
안정성의 안정성 → Node.js의 안정성

taggon commented on source/_posts/2016-08-17-announcements-cars-dynatrace.md:
```diff
@@ -0,0 +1,141 @@
+optimize Node.js.
+-->
+
+Node.js 재단은 암스테르담에서 [Node.js Interactive Europe](http://events.linuxfoundation.org/events/node-interactive-europe) 을 9월 15-18일, 11월 29일부터 12월 2일까지 텍사스 주 오스틴 (Austin, Texas)에서 [Node.js Interactive North America](http://events.linuxfoundation.org/events/node-interactive)를 개최했습니다. 이는 벤더 중립적인 이벤트로서, 앞으로 코드를 개발하는 개발자는 Node.js에 대해 깊이 있게 살펴볼 수 있습니다. Node.js 핵심 기여자는 새로운 개발에 대한 통찰력을 제공하고, 엔터프라이즈 사용자 및 벤더는 ...
```
각기 독립적인 두 개의 행사를 나타내는 거라면 문장 구조도 같은 편이 보기 편합니다.
암스테르담, Node.js Interactive Europe, 날짜 순이었으면
텍사스 주 오스틴, Node.js Interactive North America, 날짜 순으로 동등한 문장 형식으로 표현해햐 할 것입니다. 지금은 조금 헷갈리게 나열되어 있습니다.

taggon replied:
Node.js에 대해 깊이 있게 살펴본다가 아니라 Node.js 미래에 대하 살펴보는 거 아닌가요? 원문에 있는 future가 번역에 빠져있습니다.

taggon commented on source/_posts/2016-08-17-announcements-cars-dynatrace.md:
```diff
@@ -0,0 +1,141 @@
+Node.js is used by tens of thousands of organizations in more than 200 countries and amasses more than 4 million active users per month. It is the runtime of choice for high-performance, low latency applications, powering everything from enterprise applications, robots, API engines, cloud stacks and mobile websites.
+-->
+
+Node.js는 200개 이상의 국가에서 수만 개의 조직이 사용하며 매달 400만 명 이상의 사용자가 활발히 사용합니다. 고성능, 낮은 지연 시간 애플리케이션, 엔터프라이즈 애플리케이션, 로봇, API 엔진, 클라우드 스택 및 모바일 웹사이트에서 모든 것을 선택할 수 있습니다.
```
[of choice for _something_](http://endic.naver.com/enkrIdiom.nhn?sLn=kr&idiomId=2867ecb0062a4acc938c44a362138397&query=of+choice) <- ~의 용도로 선택되다. 라는 의미입니다.

taggon commented on source/_posts/2016-08-17-announcements-cars-dynatrace.md:
```diff
@@ -0,0 +1,141 @@
+application performance with the help of Dynatrace offerings.
+-->
+
+디지털 성능 관리 소프트웨어 회사인 [Dynatrace](https://www.dynatrace.com)는 Node.js 가 자사의 엔터프라이즈 고객을 위한 핵심 구성요소라고 믿고 있습니다. 더 많은 기업들이 디지털 변환을 수행하고 마이크로 서비스 유형의 아키텍처를 구현하기 시작함에 따라, Node.js는 프론트엔드 및 백엔드 시스템을 결합하여 Dynatrace 오퍼링의 도움을 받아 애플리케이션 성능을 전체적으로 보여주는 핵심기술입니다
```
오퍼링 부분이 어색한데, "제공한 것"이라는 의미가 있으니 여기서는 아예 "오퍼링"이라는 단어를 빼는 게 더 자연스러울 것 갈습니다.

yous replied:
- Node.js 가 → Node.js가
- 핵심기술입니다 → 핵심기술입니다.

아래에서 digital transformation을 디지털 혁신이라고 번역하셨는데, 여기에는 디지털 변환이라고 되어 있습니다.

yous commented on source/_posts/2016-08-17-announcements-cars-dynatrace.md:
```diff
@@ -0,0 +1,141 @@
+---
+category: articles
+title: Cars.com와 Dynatrace는 Node.js 플랫폼의 안정성과 성공을 지원하기 위해 재단에 참여합니다.
```
Cars.com와 → Cars.com과

yous commented on source/_posts/2016-08-17-announcements-cars-dynatrace.md:
```diff
@@ -0,0 +1,141 @@
+> New Node.js Foundation Members Drive Enterprise Growth
+-->
+
+> 새로운 Node.js 회원이 기업성장을 주도합니다.
```
Node.js 회원이 → Node.js 재단 회원이

yous commented on source/_posts/2016-08-17-announcements-cars-dynatrace.md:
```diff
@@ -0,0 +1,141 @@
+help fuel that growth through the Node.js Foundation.”
+-->
+
+Cars.com 소프트웨어 개발 관리자인 Darrel Pratt는 "Node.js 이전에는 Cars.com의 요구 사항을 효과적으로 충족시키지 못하는 오래된 콘텐츠 관리 솔루션을 사용하고 있었습니다. Cars.com은 매월 3,000~3,500만 건의 웹사이트 방문을 받고 있습니다."
```
Darrel → Darrell

yous commented on source/_posts/2016-08-17-announcements-cars-dynatrace.md:
```diff
@@ -0,0 +1,141 @@
+-->
+
+Cars.com 소프트웨어 개발 관리자인 Darrel Pratt는 "Node.js 이전에는 Cars.com의 요구 사항을 효과적으로 충족시키지 못하는 오래된 콘텐츠 관리 솔루션을 사용하고 있었습니다. Cars.com은 매월 3,000~3,500만 건의 웹사이트 방문을 받고 있습니다."
+"Node.js는 우리가 성장하고 마이크로 서비스 인프라로 변경할 수 있게 하는데 필요한 유틸리티를 제공합니다. 커뮤니티, 광범위한 라이브러리 및 뛰어난 도구를 통해 Node.js의 미래에는 많은 잠재력이 있으며 Node.js 재단을 통해 성적을 기여할 수 있게 되어 기쁩니다." 라고 말했습니다.
```
- 성적을 기여할 수 있게 되어 → 성장에 기여할 수 있게 되어
- "... 기쁩니다." 라고 → "... 기쁩니다."라고

yous commented on source/_posts/2016-08-17-announcements-cars-dynatrace.md:
```diff
@@ -0,0 +1,141 @@
+platform and help developers improve their monitoring capabilities.”
+-->
+
+Dynatrace 수석 기술 전략가인 Alois Reitbauer는 "디지털 혁신은 기업이 비즈니스를 영위하는 방식과 직접적으로 관련이 있으며, 기업들이 기업의 성능 모니터링 요구 사항을 직접 충족할 수 있도록 조정하는 방식입니다." 라고 말했습니다.
```
"... 방식입니다." 라고 → "... 방식입니다."라고

> “Node.js very often plays a key role in their transformation process, so we are excited to actively contribute to the Node.js platform and help developers improve their monitoring capabilities.”

위 부분의 번역이 누락되었습니다.

yous commented on source/_posts/2016-08-17-announcements-cars-dynatrace.md:
```diff
@@ -0,0 +1,141 @@
+
+Node.js 재단 커뮤니티 관리자인 Mikeal Rogers는
+"Node.js 재단에 합류함으로써 기업들은 안정성의 안정성을 높이고 광범위한 채택과 개발을 가속화하는 데 일조하고 있습니다."
+"Cars.com와 Dynatrace 같은 새로운 멤버들은 기업 사용자의 요구와 수요를 지원하는 서비스, 교육, 훈련 및 이벤트를 개발하기 위해 재단의 업무에 매우 중요합니다." 라고 말했습니다.
```
- Cars.com와 → Cars.com과
- "... 중요합니다." 라고 → "... 중요합니다."라고

yous commented on source/_posts/2016-08-17-announcements-cars-dynatrace.md:
```diff
@@ -0,0 +1,141 @@
+optimize Node.js.
+-->
+
+Node.js 재단은 암스테르담에서 [Node.js Interactive Europe](http://events.linuxfoundation.org/events/node-interactive-europe) 을 9월 15-18일, 11월 29일부터 12월 2일까지 텍사스 주 오스틴 (Austin, Texas)에서 [Node.js Interactive North America](http://events.linuxfoundation.org/events/node-interactive)를 개최했습니다. 이는 벤더 중립적인 이벤트로서, 앞으로 코드를 개발하는 개발자는 Node.js에 대해 깊이 있게 살펴볼 수 있습니다. Node.js 핵심 기여자는 새로운 개발에 대한 통찰력을 제공하고, 엔터프라이즈 사용자 및 벤더는 ...
```
- \[Node.js Interactive Europe\](...) 을 → \[Node.js Interactive Europe\](...)을
- 텍사스 주 오스틴 뒤의 괄호는 지워도 될 것 같습니다. 만약 두실 거라면 괄호 앞의 공백을 지워주세요.
- 'Node.js Core'는 'Node.js 코어'로 번역하는 게 어떨까요? Core가 일반적인 명사로 쓰인 게 아니라서 구분이 필요할 것 같습니다.

yous commented on source/_posts/2016-08-17-announcements-cars-dynatrace.md:
```diff
@@ -0,0 +1,141 @@
+The Foundation is made up of a diverse group of companies including Platinum members IBM, Intel, Joyent, Microsoft, PayPal and Red Hat. Gold members include GoDaddy and NodeSource, and Silver members include Apigee, AppDynamics, Cars.com, Codefresh, DigitalOcean, Dynatrace, Fidelity, Google, Groupon, nearForm, New Relic, npm, Opbeat, RisingStack, Sauce Labs, SAP, StrongLoop (an IBM company), Sphi ...
+-->
+
+재단은 플래티넘 멤버인 IBM, Intel, Joyent, Microsoft, PayPal 및 Red Hat을 포함한 다양한 그룹의 회사로 구성되어 있습니다. 골드 회원은 Godaddy 및 NodeSource를 포함하고 실버 회원은 Apigee, AppDynamics, Cars.com, Codefresh, DigitalOcean, Dynatrace, Fidelity, Google, Groupon, nearForm, New Relic, npm, Opbeat, RisingStack, Sauce Labs, SAP, strongLoop (IBM 회사), Sphinx, YLD, Yahoo!. 여기에 참여하십시오: [https://nodejs.org](https://nodejs.org)
```
- PayPal 및 Red Hat을 → PayPal, Red Hat을
  같은 수준의 항목을 나열할 때 and나 or를 특별히 번역하지 않으셔도 됩니다.
- strongLoop (IBM 회사) → StrongLoop(IBM 회사)
- 여기에 참여하십시오: → 여기에서 참여하세요.

> 재단은 플래티넘 멤버인 ... Red Hat을 포함한 다양한 그룹의 회사로 구성되어 있습니다. 골드 회원은 ... NodeSource를 포함하고 실버 회원은 ... Yahoo!.

재단은 플래티넘 멤버인 ... Red Hat, 골드 회원인 ... NodeSource, 실버 회원인 ... Yahoo!를 포함한 다양한 그룹의 회사로 구성되어 있습니다.

devjin0617 committed:
Update cars-dynatrace.md (#414)

devjin0617 committed:
Update cars-dynatrace.md (#414)

taggon commented on source/_posts/2016-08-17-announcements-cars-dynatrace.md:
```diff
@@ -0,0 +1,142 @@
+-->
+
+**샌프란시스코, 2016년 8월 17일**
+Node.js 플랫폼의 개발을 개선하려고 커뮤니티와 기업이 함께 만든 [Node.js 재단](https://nodejs.org/en/foundation/)은 오늘 [Cars.com](https://www.cars.com/)과 [Dynatrace](https://www.dynatrace.com)가 재단에 실버회원으로 가입했습니다.
```
가입했습니다. > 가입했다고 발표했습니다.

taggon commented on source/_posts/2016-08-17-announcements-cars-dynatrace.md:
```diff
@@ -0,0 +1,142 @@
+developer survey.
+-->
+
+기업용 도구 및 서비스 에코시스템은 Node.js를 중심으로 빠르게 융합되어 엔터프라이즈 애플리케이션, 로봇, API 엔진, 클라우드 스택, 모바일 웹사이트의 모든 것을 지원하는 범용 플랫폼으로 발전되었습니다. 자바스크립트 런타임 환경은 자원 효율, 성능이 뛰어나며 확장성에 잘 어울려 애플리케이션 개발 및 마이크로 서비스 아키텍처를 위해 스타트업, 엔터프라이즈 기업에서 점차 대체되고 있습니다. 또한 Node.js는 StackOverflow가 조사한 2016년 개발자 설문조사에서 풀스택, 프론트엔드 및 백엔드 개발자를 위한 상위 10개 언어 중 하나입니다.
```
> 기업용 도구 및 서비스 에코시스템은 Node.js를 중심으로 빠르게 융합되어 엔터프라이즈 애플리케이션, 로봇, API 엔진, 클라우드 스택, 모바일 웹사이트의 모든 것을 지원하는 범용 플랫폼으로 발전되었습니다.

**범용 플랫폼으로 발전된** 술어의 주어가 없습니다. 의미를 명확하게 하기 위해 주어 추가, 수동태를 능동태로 다음과 같이 수정할 수 있습니다(예시일 뿐이니 원하시는 대로 바꾸셔도 됩니다).

> 기업용 도구 및 서비스 에코시스템은 Node.js를 중심으로 빠르게 융합되어 Node.js는 엔터프라이즈 애플리케이션, 로봇, API 엔진, 클라우드 스택, 모바일 웹사이트의 모든 것을 지원하는 범용 플랫폼으로 발전했습니다.

taggon replied:
이런 다른 이야기인데 나중에 번역하실 때 문장 단위로 줄을 나누어 두면 리뷰를 보기가 조금 더 편합니다. ^^

taggon replied:
두 번째 문장에서 The JavaScript runtime은 문맥상 Node.js를 가리키는 걸로 보입니다. 단순히 "자바스크립트 런타임 환경"이라고만 하면 마치 전체 런타임 환경을 지칭하는 것처럼 비칠 수 있습니다. 이 부분은 영어권 사용자들이 종종 하는 주어 반복 회피를 위해 선택한 단어로 보이므로 그냥 "Node.js는 자원 효율, 성능이 뛰어나며..."로 바꾸셔도 문제가 없을 것 같습니다.

devjin0617 committed:
Update cars-dynatrace.md (#414)
